ynput/OpenPype
# TVPaint: Optional Validation plugins can be de/activated by user

## Changelog Description
Added `OptionalPyblishPluginMixin` to TVpaint plugins that can be optional.

## Additional information
I've also fixed settings of auto detect creator which did not have settings for `enabled` and fixed `"fps"` collection.

## Testing notes:
1. Open TVPaint
2. Open Publisher in TVPaint
3. Context (Options) item should have Validator plugins that can be de/activated
